README.md

```diff
@@ -2,8 +2,25 @@
 
 If you'd like to see a new example, please open an issue.
 
-- ### Go
-- ### Python
-- ### Node.js (Typescript/Javascript)
-- ### Bash with Dagger CLI and GraphQL
-- ### GraphQL queries
+## Go
+
+- [multiarch build](./go/multiarch-build/)
+- [npm build](./go/npm-build/)
+- [yarn build](./go/yarn-build/)
+- [gradle build](./go/gradle-build/)
+
+## Python
+
+- [basic example](./python/basic-example/)
+
+## Node.js (Typescript/Javascript)
+
+- [react build](./nodejs/react-build/)
+
+## Bash with Dagger CLI and GraphQL
+
+- [git build](./bash/git-build/)
+
+## GraphQL queries
+
+- TODO
```

Comment:
Could link here so folks can discover some queries and try immediately in the API Playground: https://docs.dagger.io/api/reference/